ci/docker/fuzzer/requirements.txt

```diff
@@ -24,3 +24,6 @@ six==1.16.0
 wadllib==1.3.6
 wheel==0.46.1
 zipp==3.19.1
+# for vector search stress tests
```

Comment:
Minor: It would be nicer to add both dependencies to above _sorted_ list and add two comments like this:

```
numpy==2.2.6 # for vector search stress test
```

Reply:
Hmm, I don't quite understand how the fuzzer (ci/docker/fuzzer/requirements.txt) is related to the stress test. Is this really the right file to modify?

Reply:
@shankar-iyer Since we are now using the performance-comparison image (at least based on what I see in jobs_config.py), we likely don't need the changes in fuzzer/requirements.txt.